docs/ha_3scale.adoc

```diff
@@ -217,4 +217,67 @@ In this setup, to ensure good database access latency, each cluster will have it
+
+==== Soft Databases limitation (out of 3scale scope)
+
+One of the most difficult parts for any application deployed on *active*/*active* (not only 3scale), is having databases on *active*/*active* mode, and in case of 3scale there are many different databases involved which would require different *active*/*active* implementations for each of them:
+
+- System-mysql (or system-postgres or system-oracle, you can choose)
+- System-redis
+- Backend-redis
+- Zync-database (postgresql)
+- system-sphinx (doesn't need HA)
+- system-memcached (doesn't need HA)
+
+So, from 3scale point of view, an administrator need to ensure that database connection strings configured in 3scale APIManager custom resource on any OCP cluster will be always a master instance (permitting write operations).
+
+____
+*NOTE*
+
+*This database limitation is a soft limitation because although being very complex (implementing active/active minimum for mysql, postgresql, redis is not easy), can be achieved.*
+____
+
+==== Hard Application limitation: OpenShift Routes
+
+However, the main reason why *active*/*active* can not be done although you previously deployed databases on *active*/*active*, is *System* dev/admin portals and *Apicasts* OpenShift Routes (that need to be the same on all OCP clusters) being managed by *Zync* component.
+
+This is an example flow since a client wants a *System* new dev portal and incoming traffic can eventually reach system-app pods thanks to a newly created OpenShift Route:
+
+. System-app receives the request from UI/API to create a dev/admin portal or apicast OpenShift  Route
+. System-app enqueues the background job to create those OpenShift Routes into system-redis
+. System-sidekiq takes the job from system-redis, process it and contacts with zync API (zync deployment)
+. Zync API (zync deployment) creates a background job into zync-database (postgres)
+. Zync-que takes the route-creation-job from zync-database (postgres) and create the final OpenShift  Routes into the cluster that proccessed that job
+
+When having N clusters, imagine Cluster-A and Cluster-B, any of them can receive traffic managed by either Cluster-A-system-app or Cluster-B-system-app, so they will enqueue the job into its own database, and would follow the whole procedure:
+
+```
+System-app -> system-redis -> system-sidekiq -> zync (api) -> zync-database -> zync-que -> OpenShift Routes creation
+```
+
+And finally, the OpenShift Routes will be created *only* into the cluster that processed the route-creation-job (not on both OCP clusters), so incoming traffic managed by OpenShift Router will work only on one cluster (the one with the needed OpenShift Routes), so not having *active*/*active*.
+
+____
+*NOTE*
+
+*This application limitation is a hard limitation because there is no way to solve it, since the OpenShift Routes creation need to be done on any OCP cluster working on active/active, but this OpenShift Routes creation will be done just on a single cluster.*
```

Comment:
```suggestion
*This application limitation is a hard limitation because the first worker that fetches the job, will create the router on the cluster it's running. So, each cluster will have a subset of routes. *
```